From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tobias Werth <werth@cs.fau.de>
Date: Wed, 19 Feb 2020 21:30:44 +0100
Subject: cherry-pick: Bump default script_filesize_limit.

We even warn in our very own config check when it is too low :-)

This has been the default at past NWERCs and WFs.

(cherry picked from commit 053f592d7f4a51027bf8f85af4dfe9a6510aa5ba)
---
 etc/db-config.yaml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/db-config.yaml b/etc/db-config.yaml
index be57cd7db..4e0afb305 100644
--- a/etc/db-config.yaml
+++ b/etc/db-config.yaml
@@ -77,7 +77,7 @@
             description: Maximum memory usage (in kB) by compile/compare scripts. This is a safeguard against malicious code and buggy script, so a reasonable but large amount should do.
         -   name: script_filesize_limit
             type: int
-            default_value: 540672
+            default_value: 2621440
             public: false
             description: Maximum filesize (in kB) compile/compare scripts may write. Submission will fail with compiler-error when trying to write more, so this should be greater than any *intermediate or final* result written by compilers.
         -   name: timelimit_overshoot
-- 
2.20.1

